tests/topology_test.go

```diff
@@ -885,3 +885,55 @@ func TestUserMetadata(t *testing.T) {
 
 	RunTest(t, test)
 }
+
+//TestRouteTable tests route table update
+func TestRouteTable(t *testing.T) {
+	gopath := os.Getenv("GOPATH")
+	topology := gopath + "/src/github.com/skydive-project/skydive/scripts/simple.sh"
+
+	test := &Test{
+		setupCmds: []helper.Cmd{
+			{fmt.Sprintf("%s start 124.65.91.42/24 124.65.92.43/24", topology), true},
+		},
+
+		tearDownCmds: []helper.Cmd{
+			{fmt.Sprintf("%s stop", topology), true},
+		},
+
+		checks: []CheckFunction{
+			func(c *CheckContext) error {
+				prefix := "g"
+				if !c.time.IsZero() {
+					prefix += fmt.Sprintf(".Context(%d)", common.UnixMillis(c.time))
+				}
+
+				node, err := c.gh.GetNode(prefix + ".V().Has('IPV4', '124.65.91.42/24')")
+				if err != nil {
+					return fmt.Errorf("Failed to find a node with IP 124.65.91.42/24")
+				}
+
+				routingTable := node.Metadata()["RoutingTable"].(map[string]interface{})
+				noOfRoutes := len(routingTable["Routes"].([]interface{}))
+
+				helper.ExecCmds(t,
+					helper.Cmd{Cmd: "ip netns exec vm1 ip route add 124.65.92.0/24 dev eth0", Check: true},
+					helper.Cmd{Cmd: "sleep 5", Check: false},
+				)
+
+				node, err = c.gh.GetNode(prefix + ".V().Has('IPV4', '124.65.91.42/24')")
+				routingTable = node.Metadata()["RoutingTable"].(map[string]interface{})
+				newNoOfRoutes := len(routingTable["Routes"].([]interface{}))
+
+				helper.ExecCmds(t,
+					helper.Cmd{Cmd: "ip netns exec vm1 ip route del 124.65.92.0/24 dev eth0", Check: true},
```

Comment:
for the CI error it is normal as in the replay mode the vm1 netns doesn't exist anymore. I will think about a solution to fix this design issue